site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -201,21 +212,20 @@
 
     function submitFormAjax() {
         let url = $('#peers-submit-url').val();
-        let form = $("#admin-gradeable-edit-peers-form");
         let grader_id = $("#admin-gradeable-edit-peers-title").text().substr(18);
         let fd = new FormData();
         fd.append('csrf_token', $("#csrf-token").val());
         fd.append('grader_id', grader_id);
         fd.append('remove_grader', $('#removed-all').text());
         let curr_student_ids = [];
         $(".edit_user_ids :input.peerstudents").each(function() {
-            if ($(this).val() != "") curr_student_ids.push($(this).val());
+            if ($(this).val() !== "") curr_student_ids.push($(this).val());
```

Comment:
Is this convention allowed in JS files? Although this is not checked in twig files, perhaps we should follow the same convention throughout our files.